Source/Scene/PointShading.js

```diff
@@ -0,0 +1,79 @@
+define([
+    '../Core/defaultValue'
+], function(
+    defaultValue
+) {
+    'use strict';
+
+    /**
+     * Options for performing point attenuation based on geometric error when rendering
+     * pointclouds using 3D Tiles.
+     *
+     * @param {Object} [options] Object with the following properties:
+     * {Boolean} [options.geometricErrorAttenuation=false] Perform point attenuation based on geometric error.
+     * {Number} [options.geometricErrorScale=1.0] Scale to be applied to each tile's geometric error.
+     * {Number} [options.maximumAttenuation] Maximum attenuation in pixels. Defaults to the Cesium3DTileset's maximumScreenSpaceError.
+     * {Number} [options.baseResolution] Average base resolution for the dataset in meters. Substitute for Geometric Error when not available.
+     * {Boolean} [options.eyeDomeLighting=false] When true, use eye dome lighting when drawing with point attenuation.
+     * {Number} [options.eyeDomeLightingStrength=1.0] Increasing this value increases contrast on slopes and edges.
+     * {Number} [options.eyeDomeLightingRadius=1.0] Increase the thickness of contours from eye dome lighting.
+     * @constructor
+     */
+    function PointShading(options) {
+        var pointShading = defaultValue(options, {});
+
+        /**
+         * Perform point attenuation based on geometric error.
+         * @type {Boolean}
+         * @default false
+         */
+        this.geometricErrorAttenuation = defaultValue(pointShading.geometricErrorAttenuation, false);
```

Comment:
I know I've said this online and offline, but I still think we are going with a naming mistake here not just calling this `attenuation` for the same reason that `Primitive` is just called "Primitive" and the special cases have more precise names like `GroundPrimitive`.  Always use the most obvious name for the common case.

Please reconsider here.